tests/utils.py

```diff
@@ -183,6 +183,36 @@ def prepend_tmp(path):
             freqs=[2e14, 2.5e14],
             mode_spec=ModeSpec(),
         ),
+        Near2FarAngleMonitor(
+            center=(1, 2, 3),
+            size=(2, 2, 2),
+            freqs=[250e12, 300e12],
+            name="n2f_angle",
+            custom_origin=(1, 2, 3),
+            phi=[0, np.pi / 2],
+            theta=np.linspace(-np.pi / 2, np.pi / 2, 100),
+        ),
+        Near2FarCartesianMonitor(
+            center=(1, 2, 3),
+            size=(2, 2, 2),
+            freqs=[250e12, 300e12],
+            name="n2f_cartesian",
+            custom_origin=(1, 2, 3),
+            x=[-1, 0, 1],
+            y=[-2, -1, 0, 1, 2],
+            plane_axis=2,
+            plane_distance=5,
+        ),
+        # Near2FarKSpaceMonitor(
```

Comment:
why was this removed out of curiosity?

Reply:
Oh I commented on that in the PR description, it breaks the hdf5 IO test... but this will be reorganized *again* soon so I left it for now.